bin/peggy-cli.mjs

```diff
@@ -0,0 +1,593 @@
+      )
+      .option(
+        "-T, --test-file <filename>",
+        "Test the parser with the contents of the given file, outputting the result of running the parser instead of the parser itself. If the input to be tested is not parsed, the CLI will exit with code 2"
+      )
+      .option("--trace", "Enable tracing in generated parser", false)
+      .addOption(
+        // Not interesting yet.  If it becomes so, unhide the help.
+        new Option("--verbose", "Enable verbose logging")
+          .hideHelp()
+          .default(false)
+      )
+      .addOption(
+        new Option("-O, --optimize <style>")
+          .hideHelp()
+          .argParser(() => {
+            this.print(this.std.err, "Option --optimize is deprecated from 1.2.0 and has no effect anymore.");
+            this.print(this.std.err, "It will be deleted in 2.0.");
+            this.print(this.std.err, "Parser will be generated in the former \"speed\" mode.");
+            return "speed";
+          })
+      )
+      .action((inputFile, opts) => { // On parse()
+        this.inputFile = inputFile;
+        this.argv = opts;
+
+        if (this.argv.allowedStartRules.length === 0) {
+          // [] is an invalid input, as is null
+          // undefined doesn't work as a default in commander
+          delete this.argv.allowedStartRules;
+        }
+
+        // Combine plugin/plugins
+        if ((this.argv.plugin && (this.argv.plugin.length > 0))
+            || (this.argv.plugins && (this.argv.plugins.length > 0))) {
+          this.argv.plugins = [
+            ...(this.argv.plugins || []),
+            ...(this.argv.plugin || []),
+          ].map(val => {
+            if (typeof val !== "string") {
+              return val;
+            }
+            // If this is an absolute or relative path (not a module name)
+            const id = (path.isAbsolute(val) || /^\.\.?[/\\]/.test(val))
+              ? path.resolve(val)
+              : val;
+            let mod = null;
+            try {
+              mod = require(id);
+            } catch (e) {
+              if (e.code !== "MODULE_NOT_FOUND") { throw e; }
+              this.error(`Requiring "${id}": ${e.message}`);
+            }
+            return mod;
+          });
+        }
+        delete this.argv.plugin;
+
+        // Combine dependency/dependencies
+        this.argv.dependencies = this.argv.dependencies || {};
+        if (this.argv.dependency) {
+          if (this.argv.dependency.length > 0) {
+            for (const dep of this.argv.dependency) {
+              const [name, val] = dep.split(":");
+              this.argv.dependencies[name] = val ? val : name;
+            }
+          }
+          delete this.argv.dependency;
+        }
+
+        if ((Object.keys(this.argv.dependencies).length > 0)
+            && (MODULE_FORMATS_WITH_DEPS.indexOf(this.argv.format) === -1)) {
+          this.error(`Can't use the -d/--dependency option with the "${this.argv.format}" module format.`);
```

Comment:
Add also mention of `-D/--dependencies` option

Reply:
Done.